satellite/metainfo/metainfo.go

```diff
@@ -336,8 +339,13 @@ func (endpoint *Endpoint) createOrderLimitsForSegment(ctx context.Context, point
 			Limit:              orderLimit,
 			StorageNodeAddress: node.Address,
 		})
+	}
 
+	if len(limits) < redundancy.RequiredCount() {
+		err = Error.New("not enough nodes available. Available: %d, Required: %d", len(limits), redundancy.RequiredCount())
```

Comment:
Maybe change the error message to `"not enough nodes available: got %d, required %d"`